ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -464,7 +464,7 @@ const OmimSegments = styled(Segment.Group).attrs({ size: 'tiny', horizontal: tru
   }
 `
 
-const getDetailSections = (configs, gene, compact, labelProps, individualGeneData) => configs.map(
+const getDetailSections = (configs, gene, compact, labelProps, individualGeneData, noExpand) => configs.map(
```

Comment:
Add the `noExpand` parameter to disable the `expandedDisplay` config for the OMIM details. Not sure if the logic is correct or not.